payara-common/src/main/java/fish/payara/arquillian/container/payara/PayaraVersion.java

```diff
@@ -83,7 +96,7 @@ public boolean isMoreRecentThan(PayaraVersion minimum) {
                     vPart--;
                 }
 
-                if (vPart > minPart) {
+                if (vPart >= minPart) {
```

Comment:
Are there any tests for this behavior? Because I don't believe it's correct.
With this change 5.191 will be more recent than 5.193, because both are version 5.

Reply:
Made a small change to get around this - PayaraVersionTests class has been run and passes as expected with original functionality

Reply:
Reverted change to original state - overlooked end part of function